presto-product-tests/src/main/java/io/prestosql/tests/hive/TestCsv.java

```diff
@@ -164,6 +172,24 @@ private void testCreatePartitionedCsvTableAs(String tableName, String additional
         query("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCreateCsvFileSkipHeader()
+    {
+        assertThat(query("SELECT * FROM csv_with_header")).containsOnly(row("10", "value2"));
```

Comment:
This could be easily refactored to tempto convention tests.

Have you seen:./src/main/resources/sql-tests/testcases/tables_with_header_and_footer.sql?

This way it is able to change tests and then execute them without recompiling.